code_quality_tools/pmd.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
```

Comment:
I could live with these in the root:
```
/pmd.xml
/checkstyle.xml
/findbugs.xml
```
I'd create a separate dir only if you have like many extra files with odd filenames with underscores or something.